docs/Concepts/Pruning.md

```diff
@@ -1,25 +1,46 @@
 ---
-description: Pruning
+description: Node type and pruning
 ---
 
-# Pruning
+# Node type and pruning
```

Comment:
This topic covers pruning and fast sync and their impact on node types. So I think _Full and archive node types_ might be a clearer title

Reply:
Done.